src/app/_components/TextArea.js

```diff
@@ -0,0 +1,33 @@
+'use client';
+
+import PropTypes from 'prop-types';
+
+export function TextArea({
+  value, onChange, maxLength, placeholder,
+}) {
+  return (
+    <div className="flex h-full flex-col gap-0.5">
+      <textarea
+        className="appereance-none grow resize-none rounded-xl border-none text-gray-900"
+        value={value}
+        maxLength={maxLength}
+        onChange={e => onChange(e.target.value)}
+        placeholder={placeholder}
+      />
+      {maxLength && (
+        <div className="flex flex-row-reverse">
+          <span className="text-primary-500">
+            {String(value).length} / {maxLength}
+          </span>
+        </div>
+      )}
```

Comment:
![image](https://github.com/keenethics/mayak/assets/40301543/4c129378-cbb7-46db-aaa8-709c97e5d6ca)
🔴Please add the default text here, when the textarea is empty.

Reply:
thanks, done